contracts/wrapper/NameWrapper.sol

```diff
@@ -473,10 +486,9 @@ contract NameWrapper is
      */
     modifier operationAllowed(bytes32 node, uint96 fuseMask) {
         (, uint96 fuses) = getData(uint256(node));
-        require(
-            fuses & fuseMask == 0,
-            "NameWrapper: Operation prohibited by fuses"
-        );
+        if (fuses & fuseMask != 0) {
+            revert OperationProhibited(node);
```

Comment:
Perhaps this should log the fuses that were problematic (eg, `fuses & fuseMask`).